src/ui/components/services/storage.ts

```diff
@@ -0,0 +1,21 @@
+import Todo from '../models/todo';
```

Comment:
yea, that bit of code I was missing.

We could override that in `config/environment.js`, couldn't we?